src/middlewares/index.ts

```diff
@@ -10,6 +11,6 @@ import {UIOptionsObject} from '../types';
  * @return {GenericStoreEnhancer} - The redux middleware.
  */
 const middleware = (player: any, config: UIOptionsObject): any =>
-  applyMiddleware<any, any>(loggerMiddleware(config), eventDispatcherMiddleware(player));
+  applyMiddleware<any, any>(loggerMiddleware(config), eventDispatcherMiddleware(player), isPlayerSmallSizeMiddleware());
```

Comment:
how about calling it DebounceMiddleware so it can be ready in case we want to add more actions like this ?

Reply:
done